src/database/models/mod_item.rs

```diff
@@ -1,13 +1,10 @@
 use crate::database::models::team_item::Team;
-use crate::database::models::Item;
-use crate::database::Result;
-use bson::{Bson, Document};
 use serde::{Deserialize, Serialize};
 
 #[derive(Deserialize, Serialize)]
 pub struct Mod {
     /// The ID for the mod, must be serializable to base62
-    pub id: i32,
+    pub id: i64,
```

Comment:
We should create wrapper types for the different types of Ids, so that they don't get mixed up.  Also, the `version_ids` field below must use `i64` or a wrapped `i64`. (For some reason I can't comment on that line).